cross/java-11-openjdk/Makefile

```diff
@@ -82,12 +82,11 @@ COMPILE_MAKE_OPTIONS += product-images
 
```

Comment:
You are still missing `COMPILE_MAKE_OPTIONS += JOBS=$(NCPUS)` to enable parallel build based on how openjdk uses it.